plugins/obs-ffmpeg/obs-ffmpeg-vaapi.c

```diff
@@ -247,7 +244,7 @@ static bool vaapi_update(void *data, obs_data_t *settings, bool hevc)
 	int bf = (int)obs_data_get_int(settings, "bf");
 	int qp = rc_mode->qp ? (int)obs_data_get_int(settings, "qp") : 0;
 
-	av_opt_set_int(enc->context->priv_data, "qp", qp, 0);
+	enc->context->global_quality = enc->codec == CODEC_AV1 ? qp * 4 : qp;
```

Comment:
Quality range for AV1 is up to 255, unlike H264/H265 where it is 51. The same multiply by 4 is done in nvenc plugin too.

It should probably also increase maximum value to `63` here (as nvenc does), because now the maximum QP that can be set is 204.